### PR TITLE
Refactor travis file

### PR DESCRIPTION
**Pull request by lucasmoura**
Create scripts to reuse on similar integration tests scenarios on travis yaml

### CODE REVIEW

**Review by blackboxsw — CHANGES_REQUESTED** [review]
Let's move these scripts into the existing ./tools  subdir as we may at somepoint what to use and package a ./bin directory  in the deb package. I don't want to confuse tooling with releasable content..

File: bin/base_travis_integration_tests.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+create_pr_tar_file() {
+  cd $TRAVIS_BUILD_DIR/..
+  tar -zcf pr_source.tar.gz ubuntu-advantage-client
+  cp pr_source.tar.gz /tmp
+  ls -lh /tmp
+  cd $TRAVIS_BUILD_DIR
+}
+
+copy_deb_packages
```
Comment (blackboxsw) [inline bin/base_travis_integration_tests.sh:22]: How we have side-effect functions being called just by sourcing this directory. I'd like to see all explicit call-sites in the scripts which source it. So we are not suprised by hidden behavior

**Comment by lucasmoura** [comment]
@blackboxsw updated

**Review by blackboxsw — APPROVED** [review]
This looks much cleaner thank you.

**Pull request merged**